ngx-tools/src/README.md

```diff
@@ -431,7 +432,28 @@ isString('foo'); // Returns: true
 isString({}); // Returns: false
 ```
 
+### `updateControlOnInputChanges`
 
+Helper function to set value to a form control if an input has changed.
+
+```typescript
+import { SimpleChanges } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { updateControlOnInputChanges } from '@terminus/ngx-tools';
+
+...
+  @Input
+  public myInput;
+  @Input
+  public myFormControl
+
+  ngOnChanges(changes: SimpleChanges) {
+    // This will verify that whether an input has been changed. If it's changed
+    // and there is a form control, form control's value will be updated to latest value
+    updateControlOnInputChanges(changes, 'myInput', 'myFormControl'))
```

Comment:
Two last tweaks here..

- Missing semicolon on line 448.
- `'myformControl'` should be unquoted `this.myFormControl`

Reply:
I had a missing semicolon on line 453 too 😄